pulp_rpm/tests/functional/api/test_retention_policy.py

```diff
@@ -136,6 +136,9 @@ def versions_for_packages(self, packages):
         for package in packages:
             packages_by_version[package['name']].append(package['version'])
 
+        for name in packages_by_version.keys():
+            packages_by_version[name] = sorted(packages_by_version[name])
```

Comment:
I think you could just iterate over the "values" and sort the lists in place

Reply:
Hm, good point, would save some memory.